README.md

```diff
@@ -13,7 +13,45 @@ encourage more programmers to become a part of the Haskell community.
 
 ## Prerequisites (what you need to have locally)
 
-Coming Soon
+For the project to build, you will need to have `libpq-dev` installed on you computer. You can install it by running the command `sudo apt install libpq-dev`.
+
+You will also need to setup Postgres on your computer. Here are the instructions for doing so:
+
+1. Run the folowing commands with `<username>` replaced by a username of your choice.
+
+   ```
+   $ sudo apt install postgresql postgresql-contrib
+   $ sudo service postgres start
+   $ sudo -u postgres psql
```

Comment:
@rashadg1030 Thanks for writing all these instructions. But we actually don't need all of that. Our DB is shipped within `docker` container. It's already configured and works on both Linux and macOS (not sure about Windows). So developers don't need to install, configure and run the database itself in order to build, test or run `issue-wanted`. All you need is to have `docker` installed and then just run:

* `make postgres`

This command will run the database. To build `issue-wanted` you still need `libpq-dev`, so you can provide a command to install it. But that's all. So, prerequisites for `issue-wanted`:

* `ghc`
* `cabal` or `stack`
* `docker`
* `libpq-dev` (`sudo apt install libpq-dev`)